src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal-model.ts

```diff
@@ -1,12 +1,24 @@
-import { FeedbackQuestion } from '../../../../types/api-output';
+import { FeedbackQuestion, FeedbackSession } from '../../../../types/api-output';
+import { SortBy, SortOrder } from '../../../../types/sort-properties';
+
+/**
+ * The model for a feedback session tab.
+ */
+export interface FeedbackSessionTabModel {
+  feedbackSession: FeedbackSession;
+  questionsTableRowModels: QuestionToCopyCandidate[];
+  questionsTableRowModelsSortBy: SortBy;
+  questionsTableRowModelsSortOrder: SortOrder;
+
+  hasQuestionsLoaded: boolean;
+  isTabExpanded: boolean;
+  hasLoadingFailed: boolean;
+}
 
 /**
  * The model for a question to copy.
  */
 export interface QuestionToCopyCandidate {
-  courseId: string;
-  feedbackSessionName: string;
   question: FeedbackQuestion;
```

Comment:
I feel like we dont have to hold the whole `question`, just its identifiers should be enough

Reply:
The question details are used when creating the copied question for the feedback session (`instructor-session-edit-page.component.ts`, lines 930-947), so I think it would be good to continue storing the entire `FeedbackQuestion` so that the details can be easily retrieved.

Reply:
I see, then we can keep this as it is